.github/workflows/ci.yml

```diff
@@ -209,7 +206,7 @@ jobs:
         uses: changesets/action@v1
         with:
           publish: yarn changeset publish
-          commit: '[ci] release'
+          commit: 'chore: release'
           title: '[ci] release'
```

Comment:
Kept the PR title `[ci] release` but changed the commit name to `chore: release`